### PR TITLE
[BULK] DocuTune - Fix build validation issues: docs-link-absolute (part 4)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by jmart1428** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@sdwheeler

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

File: docset/winserver2016-ps/defender/Get-MpPreference.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Get-MpPreference [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
 ```
 
 ## DESCRIPTION
-The **Get-MpPreference** cmdlet gets preferences for the Windows Defender scans and updates. For more information about the preferences that this cmdlet retrieves, see [Windows Defender Preferences Class](https://docs.microsoft.com/previous-versions/windows/desktop/legacy/dn455323(v=vs.85))
+The **Get-MpPreference** cmdlet gets preferences for the Windows Defender scans and updates. For more information about the preferences that this cmdlet retrieves, see [Windows Defender Preferences Class](/previous-versions/windows/desktop/legacy/dn455323(v=vs.85))
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Get-MpPreference.md:23]:
```suggestion
The **Get-MpPreference** cmdlet gets preferences for the Windows Defender scans and updates. For more information about the preferences that this cmdlet retrieves, see [Windows Defender Preferences Class](/previous-versions/windows/desktop/legacy/dn455323(v=vs.85)).
```

File: docset/winserver2016-ps/international/Set-WinDefaultInputMethodOverride.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## INPUTS
 
 ### InputTIP
-You can pipe a locale ID and a keyboard language ID (KLID). See [Default Input Profiles (Input Locales) in Windows](https://docs.microsoft.com/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs)
+You can pipe a locale ID and a keyboard language ID (KLID). See [Default Input Profiles (Input Locales) in Windows](/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs)
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/international/Set-WinDefaultInputMethodOverride.md:59]:
```suggestion
You can pipe a locale ID and a keyboard language ID (KLID). See [Default Input Profiles (Input Locales) in Windows](/windows-hardware/manufacture/desktop/default-input-locales-for-windows-language-packs).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @alexbuckgit, a few minor additional changes, please.

File: docset/winserver2016-ps/remotedesktop/Set-RDCertificate.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ PS C:\>$Password = ConvertTo-SecureString -String "Cups34Horses&&" -AsPlainText
 PS C:\>Set-RDCertificate -Role RDRedirector -ImportPath "C:\Certificates\Redirector07.pfx" -Password $Password -ConnectionBroker "RDCB.Contoso.com"
 ```
 
-The first part of the example uses the **ConvertTo-SecureString** cmdlet to create a secure string based on a string that the user supplies and stores it in the **$Password** variable. For more information, see [ConvertTo-SecureString](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/ConvertTo-SecureString?view=powershell-6). You can also enter the `Get-Help ConvertTo-SecureString` cmdlet into your PowerShell window.
+The first part of the example uses the **ConvertTo-SecureString** cmdlet to create a secure string based on a string that the user supplies and stores it in the **$Password** variable. For more information, see [ConvertTo-SecureString](/powershell/module/microsoft.powershell.security/ConvertTo-SecureString?view=powershell-6). You can also enter the `Get-Help ConvertTo-SecureString` cmdlet into your PowerShell window.
```
Comment (sdwheeler) [inline docset/winserver2016-ps/remotedesktop/Set-RDCertificate.md:49]:
```suggestion
The first part of the example uses the **ConvertTo-SecureString** cmdlet to create a secure string based on a string that the user supplies and stores it in the **$Password** variable. For more information, see [ConvertTo-SecureString](/powershell/module/microsoft.powershell.security/ConvertTo-SecureString). You can also enter the `Get-Help ConvertTo-SecureString` cmdlet into your PowerShell window.
```

File: docset/winserver2016-ps/remotedesktop/Set-RDCertificate.md [hunk 4]
```diff
@@ -196,4 +196,4 @@ For more information, see [about_CommonParameters](https://docs.microsoft.com/po
 
 [New-RDCertificate](./New-RDCertificate.md)
 
-[about_CommonParameters](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6)
+[about_CommonParameters](/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-6)
```
Comment (sdwheeler) [inline docset/winserver2016-ps/remotedesktop/Set-RDCertificate.md:201]:
```suggestion
[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216)

```

File: docset/winserver2016-ps/scheduledtasks/Register-ScheduledTask.md [hunk 5]
```diff
@@ -342,7 +342,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](/powershell/module/microsoft.powershell.core/about/about_commonparameters).
```
Comment (sdwheeler) [inline docset/winserver2016-ps/scheduledtasks/Register-ScheduledTask.md:345]:
```suggestion

This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
-InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
-WarningAction, and -WarningVariable. For more information, see
[about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]

**Comment by alexbuckgit** [comment]
@JohanFreelancer9 @sdwheeler -

The request changes have been accepted. Please merge, thanks!

**Review by sdwheeler — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**